### ISSUE TITLE
Import Account/Region error: CloudFormation stack fails on ConfigBucketPolicy - This policy contains invalid Json 

### ISSUE THREAD

**Issue by craigjam** [awslabs/aws-perspective#24]
**Describe the bug**
CloudFormation Stack for importing another account/region fails, and then fails to roll back

**To Reproduce**
Set up AWS Perspective.
Import another region from the same account, run the CloudFormation template

**Expected behavior**
CloudFormation stack to create successfully

**Screenshots**
![image](https://user-images.githubusercontent.com/16560180/94026517-ace5c700-fdb1-11ea-8110-d33a87ef9a9b.png)

**Browser (please complete the following information):**

- Name [e.g. chrome, firefox]

**Additional context**

**Comment by craigjam** [comment]
I think this is (part of) the CloudTrail record for the failed call, so the policy was blank?

    "errorMessage": "This policy contains invalid Json",
    "requestParameters": {
        "bucketPolicy": {},
        "bucketName": "aws-perspective-configbucket-qaq1ed68eo44",
        "Host": "aws-perspective-configbucket-qaq1ed68eo44.s3.eu-west-2.amazonaws.com",
        "policy": ""
    },

**Comment by thebutler12** [comment]
Hi, if you go to the CloudFormation console and click on this stack. You should be able to see the template under the 'Template' tab. Is that blank?

**Comment by thebutler12** [comment]
After some investigation, we found that this bug occurs when importing an account that does not already have AWS Config set up (or when you select **No** for the **AlreadyHaveConfigSetup** CloudFormation parameter) It is caused by invalid JSON in the ConfigBucketPolicy. The section that is causing the problem is

```
"Sid": "HttpsOnly"
"Action": "*"
"Effect": "Deny",
"Resource": "arn:aws:s3:::${ConfigBucket}/*",
"Principal": '*'
"Condition": {
   "Bool": {
        "aws:SecureTransport": "false"
    }
}
```

As a workaround you can edit the CloudFormation template before deploying. It should look like this

```
  "Sid": "HttpsOnly",
  "Action": "*",
  "Effect": "Deny",
  "Resource": "arn:aws:s3:::${ConfigBucket}/*",
  "Principal": "*",
  "Condition": {
    "Bool": {
      "aws:SecureTransport": "false"
    }
  }
```

We are working on a fix for this and will update this ticket when it is available.

**Comment by thebutler12** [comment]
An update on this one...We have released v1.0.1 that fixes this issue. If you want you can tear down the v1.0.0 stack and replace with v1.0.1. 

If that is going to be a pain (which we understand completely), then we are working on a set of instructions to upgrade. These aren't available yet as we are still working them out, but will update the README when they are complete.

**Comment by ConnorKirk** [comment]
#86

**Comment by mohsanjaffery** [comment]
Fixed in v1.0.1
